sdk/cognitiveservices/cognitiveservices-customvision-prediction/src/predictionAPIClient.ts

```diff
@@ -17,12 +17,12 @@ import { PredictionAPIClientContext } from "./predictionAPIClientContext";
 class PredictionAPIClient extends PredictionAPIClientContext {
   /**
    * Initializes a new instance of the PredictionAPIClient class.
-   * @param apiKey API key.
    * @param endpoint Supported Cognitive Services endpoints.
+   * @param credentials Subscription credentials which uniquely identify client subscription.
    * @param [options] The parameter options
    */
-  constructor(apiKey: string, endpoint: string, options?: msRest.ServiceClientOptions) {
-    super(apiKey, endpoint, options);
+  constructor(credentials: msRest.ServiceClientCredentials, endpoint: string, options?: msRest.ServiceClientOptions) {
```

Comment:
Looks like the only change to the prediction package is that the constructor has changed. We take credentials instead of api key, but the sample in the readme is still using api key

Do we know what triggered this change? Is it a swagger change or change in the code generator?

Reply:
it is a swagger change